kano_qt_apps/lib/includes/q_app_list.h

```diff
@@ -17,7 +17,7 @@
 #include <QList>
 #include <QVariant>
 #include <QMetaType>
-#include <QQmlParserStatus>
+#include <QtQml/QQmlParserStatus>
```

Comment:
Don't think this is really needed.. How did it compile before?

Reply:
👍 The docs only require [`<QQmlParserStatus>`](http://doc.qt.io/qt-5/qqmlparserstatus.html)